safe/gui/tools/shake_grid/shakemap_converter_dialog.py

```diff
@@ -111,6 +111,25 @@ def on_input_path_textChanged(self):
             self.get_output_from_input()
         self.update_warning()
 
+    # noinspection PyPep8Naming
+    def on_input_place_textChanged(self):
+        """Action when input place layer name is changed."""
+        if os.path.exists(self.input_place.text()):
+            try:
+                self.place_layer = QgsVectorLayer(
+                    self.input_place.text(),
+                    'Nearby Cities',
+                    'ogr'
+                )
+                if self.place_layer.isValid():
+                    LOGGER.debug('Get field information')
+                    self.name_field.setLayer(self.place_layer)
+                    self.population_field.setLayer(self.place_layer)
+                else:
+                    LOGGER.debug('failed to set name field')
+            except:
```

Comment:
Can you give comment here why do you pass the exception? Also it's a good practice to use specific exception class, so that we know what exception that we catch.

Reply:
I pass it because I don't have any idea yet on what I will do :sweat_smile: 
I will include it in my next commit. do you have any suggestion for me?